src/apps/balancer-v2/helpers/balancer-v2.cache-manager.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@nestjs/common';
+import Axios from 'axios';
+
+import { Network } from '~types/network.interface';
+
+// What in the cinnamon toast fuck is this? Gather around children...
+// 1. Balancer maintains off-chain claimable amount snapshots in the JSON files described by the `manifestUrl` property
+// 2. Originally, Balancer used the `legacyMerkleRedeemAddress` to maintain claim states
+// 3. Now, Balancer uses the `merkleOrchardAddress` to maintain claim states (after `weekStart`)
+// 4. Gather the states from both to retrieve the full claimable amount state.
+
+// Resources:
+// Legacy merkle redeem configurations can be found in https://github.com/balancer-labs/erc20-redeemable
+// New merkle orchard configurations can be found in https://github.com/balancer-labs/frontend-v2
+
+export type ClaimableTokenConfig = {
```

Comment:
Delete unused type

Reply:
I'll move it next to it's utilization in `balancer-v2.claimables.config.ts`